recipe/build.sh

```diff
@@ -1,5 +1,5 @@
 mkdir build
 cd build
 export ISISROOT=$PWD
-cmake -GNinja -DJP2KFLAG=ON -Dpybindings=OFF -DbuildTests=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PREFIX ../isis
+cmake -GNinja -DJP2KFLAG=ON -Dpybindings=OFF -DKAKADU_INCLUDE_DIR=/isisData/kakadu -DbuildTests=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PREFIX ../isis
```

Comment:
Not sure if we should remove "DKAKADU_INCLUDE_DIR" for releases and just keep it in dev branch since the path is hardcoded for Jenkins.